src/main/java/org/jabref/gui/preferences/external/ExternalTabViewModel.java

```diff
@@ -227,4 +228,10 @@ public BooleanProperty useCustomFileBrowserProperty() {
     public StringProperty customFileBrowserCommandProperty() {
         return this.customFileBrowserCommandProperty;
     }
+
+    // Reset Cite Command
+
+    public void resetCiteCommandToDefault() {
+        this.citeCommandProperty.setValue(preferences.getDefaults().get(JabRefPreferences.CITE_COMMAND).toString());
```

Comment:
I would be better to have this encapsulated  in the ExternalApplicationPreferences, e.g. a method
getDefaultCitecommand.  Have a look at the PreviewPreferences to see how it's done there

Reply:
Is probably not a blocker for this PR since this change will anyway be part of a future PR improving all the preferences objects carry their own defaults.

Reply:
Just read this. I still changed both requested Changes